coffea/processor/accumulator.py

```diff
@@ -72,8 +75,9 @@ def iadd(a: Accumulatable, b: Accumulatable) -> Accumulatable:
         lhs, rhs = set(a), set(b)
         for key in lhs & rhs:
```

Comment:
Why not loop through `a` and then `b` here as was done for `add()`?

Reply:
Okay, I changed it. The result is the same. This only makes the code look more consistent.